content/blog/customer-managed-agents-kubernetes/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Kubernetes-native Option for Pulumi Deployments Customer-Managed Agents"
+allow_long_title: true
+date: 2024-07-29T09:08:31-07:00
+draft: false
+meta_desc: Introducing Kubernetes-native support for Customer Managed Agents in Pulumi Cloud for enhanced flexibility and control over your infrastructure deployments.
+meta_image: meta.png
+authors:
+    - levi-blackstone
+    - meagan-cojocar
+tags:
+    - releases
+---
+
+We are excited to announce the availability of Kubernetes-native support for Customer-Managed Agents for Pulumi Deployments, further enhancing the flexibility and control over your infrastructure deployments. This new feature allows you to self-host deployment agents within your Kubernetes environment, bringing the same power and flexibility of Pulumi-hosted deployments to isolated environments.
+
+## Customer Managed Agents: Flexibility and Control
+
+Customer Managed Agents, [announced earlier this year](/blog/customer-managed-deployment-agents-launch), allow you to self-host deployment agents, delivering the power and flexibility of Pulumi Deployments within your own infrastructure. Here’s how you can benefit:
+
+- **Host Anywhere**: Deploy Customer Managed Agents on any hardware or environment of your choice, including fully private VPCs.
+- **Compliance and Security**: Keep your cloud provider credentials within your private network, meeting stringent compliance requirements.
+- **Scalability**: Set up multiple agent pools and scale dynamically, supporting up to 150 concurrent deployments.
+- **Versatility**: Mix and match Pulumi-hosted and self-hosted deployments to suit different infrastructure needs.
+
+### Key Features
+
+1. **Multiple Deployment Triggers**: Supports all Pulumi deployment triggers such as Drift Detection, Scheduled Deployments, click to deploy, REST API, Git push, Review Stacks, and remote Automation API.
+2. **OpenID Authentication**: Fetch tokens dynamically for increased security.
+3. **Flexible Configuration**: Configure through YAML files or environment variables, adaptable to various deployment environments.
+
+### Docker and Kubernetes Support
```

Comment:
While I agree with reiterating the value proposition, this benefits/key features seems to bury the lead and push the actual change quite far down the page.